app/controllers/admins/nonprofit_applications_controller.rb

```diff
@@ -8,5 +8,15 @@ def show
     def index
       @nonprofit_applications = NonprofitApplication.all
     end
+
+    def destroy
+      application = NonprofitApplication.find(params[:id])
+      if application.delete
+        flash[:success] = "Successfully deleted #{application.nonprofit.organization_name}'s application."
```

Comment:
You can just use `application.organization_name` which is equivalent (delegated in https://github.com/calblueprint/calblueprint.org/blob/master/app/models/nonprofit_application.rb#L43).

To use i18n for this message you wanna do something like http://guides.rubyonrails.org/i18n.html#passing-variables-to-translations